make/CreateJmods.gmk

```diff
@@ -169,7 +169,7 @@ ifeq ($(MODULE), java.base)
     endif
   endif
 else # not java.base
-  ifeq ($(call isTargetOs, windows), true)
+  ifeq ($(call isTargetOs, windows)+$(CREATING_BUILDJDK), true+false)
```

Comment:
If we are creating a buildjdk (which really means we are cross-compiling), we must not meddle with the Windows runtime dll:s, since they are for the target CPU architecture. To be *fully* correct, we should really turn the Windows runtime dll:s into a "normal"/BUILD_ split version, like with SYSROOT etc. 

But since this is just used locally for building, I'm assuming the build system has these runtime DLLs installed already. There seems to be no point in wasting time preparing a buildjdk with files that are just bundled for installation.

Maybe there's a better way to express this, but the only thing I could get a hold of for the fact that we are creating a build-JDK (that is, not targeting the normal OPENJDK_TARGET_CPU) was the CREATING_BUILDJDK variable.

Reply:
I reverted this change, and added a comment when we copy the libs.